### PR TITLE
Removed formatting leftovers.

### PR DESCRIPTION
**Pull request by matejak**
Removed italics="..." and config_item="..." from where it was applicable.

Fixes: #3053

### CODE REVIEW

File: firefox/guide/benchmark.yml [hunk 1]
```diff
@@ -10,21 +10,21 @@ description: |
...
     {{{ weblink(link="https://www.open-scap.org/security-policies/scap-security-guide") }}}.
     <br/><br/>
     Providing system administrators with such guidance informs them how to securely
     configure systems under their control in a variety of network roles. Policy
     makers and baseline creators can use this catalog of settings, with its
     associated references to higher-level security control catalogs, in order to
-    assist them in security baseline creation. This guide is a italics="catalog, not a
-    checklist," and satisfaction of every item is not likely to be possible or
+    assist them in security baseline creation. This guide is a <em>catalog, not a
+    checklist</em> and satisfaction of every item is not likely to be possible or
```
Comment (yuumasato) [inline firefox/guide/benchmark.yml]: A comma was dropped here.

File: fuse6/guide/benchmark.yml [hunk 2]
```diff
@@ -10,21 +10,21 @@ description: |
...
     {{{ weblink(link="https://www.open-scap.org/security-policies/scap-security-guide") }}}.
     <br/><br/>
     Providing system administrators with such guidance informs them how to securely
     configure systems under their control in a variety of network roles. Policy
     makers and baseline creators can use this catalog of settings, with its
     associated references to higher-level security control catalogs, in order to
-    assist them in security baseline creation. This guide is a italics="catalog, not a
-    checklist," and satisfaction of every item is not likely to be possible or
+    assist them in security baseline creation. This guide is a <em>catalog, not a
+    checklist</em> and satisfaction of every item is not likely to be possible or
```
Comment (yuumasato) [inline fuse6/guide/benchmark.yml]: Dropped comma.

File: linux_os/guide/benchmark.yml [hunk 3]
```diff
@@ -10,21 +10,21 @@ description: |
...
     {{{ weblink(link="https://www.open-scap.org/security-policies/scap-security-guide") }}}.
     <br/><br/>
     Providing system administrators with such guidance informs them how to securely
     configure systems under their control in a variety of network roles. Policy
     makers and baseline creators can use this catalog of settings, with its
     associated references to higher-level security control catalogs, in order to
-    assist them in security baseline creation. This guide is a italics="catalog, not a
-    checklist," and satisfaction of every item is not likely to be possible or
+    assist them in security baseline creation. This guide is a <em>catalog, not a
+    checklist</em> and satisfaction of every item is not likely to be possible or
```
Comment (yuumasato) [inline linux_os/guide/benchmark.yml]: Dropped comma.

File: wrlinux/guide/benchmark.yml [hunk 4]
```diff
@@ -10,21 +10,21 @@ description: |
...
     {{{ weblink(link="https://www.open-scap.org/security-policies/scap-security-guide") }}}.
     <br/><br/>
     Providing system administrators with such guidance informs them how to securely
     configure systems under their control in a variety of network roles. Policy
     makers and baseline creators can use this catalog of settings, with its
     associated references to higher-level security control catalogs, in order to
-    assist them in security baseline creation. This guide is a italics="catalog, not a
-    checklist," and satisfaction of every item is not likely to be possible or
+    assist them in security baseline creation. This guide is a <em>catalog, not a
+    checklist</em> and satisfaction of every item is not likely to be possible or
```
Comment (yuumasato) [inline wrlinux/guide/benchmark.yml]: Dropped comma.

**Comment by matejak** [comment]
Good catch, fixed by rebase.

**Pull request merged**